source/SkiaSharp.Views.WinUI/SkiaSharp.Views.WinUI/SkiaSharp.Views.WinUI.csproj

```diff
@@ -5,6 +5,8 @@
     <RootNamespace>SkiaSharp.Views.Windows</RootNamespace>
     <AssemblyName>SkiaSharp.Views.Windows</AssemblyName>
     <PackagingGroup>SkiaSharp.Views.WinUI</PackagingGroup>
+    <DefineConstants>$(DefineConstants);WINDOWS</DefineConstants>
+    <PublishTrimmed>false</PublishTrimmed>
```

Comment:
Do we need this?

Reply:
Also, isn't WINDOWS already defined?  Did you mean WINUI instead? How does this (if at all) relate to the || WINUI added in the cs?

Reply:
The PublishTrimmed is scattered thought... is that needed as I think it is an app feature.